sdk-http-vertx/src/main/java/dev/restate/sdk/http/vertx/RequestHttpServerHandler.java

```diff
@@ -86,6 +87,22 @@ public void handle(HttpServerRequest request) {
       return;
     }
 
+    // check protocol version
+    final String protocolVersionString = request.getHeader(CONTENT_TYPE);
+
+    final Protocol.ServiceProtocolVersion serviceProtocolVersion =
+        ServiceProtocol.parseServiceProtocolVersion(protocolVersionString);
+
+    if (!ServiceProtocol.is_supported(serviceProtocolVersion)) {
+      final String errorMessage =
+          String.format(
+              "Service endpoint does not support the service protocol version '%s'.",
+              protocolVersionString);
+      LOG.warn(errorMessage);
+      request.response().setStatusCode(UNSUPPORTED_MEDIA_TYPE.code()).end(errorMessage);
```

Comment:
add content type "text/plan", same below

Reply:
Will add it.